tensorflow/lite/experimental/micro/arduino-esp32/debug_log.cc

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/lite/experimental/micro/debug_log.h"
+
+#include "Arduino.h"
+
+// The Arduino DUE uses a different object for the default serial port shown in
```

Comment:
Since this implementation is specific for arduino-esp32 we can remove the `DEBUG_SERIAL_OBJECT` macro: __SAM3X8E__ will only be defined if we are using the Arduino DUE.